Lib/enum.py

```diff
@@ -577,22 +577,24 @@ def __new__(metacls, cls, bases, classdict, *, boundary=None, _simple=False, **k
                         attributes -- see the documentation for details.
                         """)
             else:
-                member = list(enum_class)[0]
-                enum_length = len(enum_class)
+                # avoid list(enum_class) because that doesn't include aliases
```

Comment:
Aliases should not be included, and are not considered part of the length of the `Flag`.

This issue will need to be fixed a different way.

Reply:
Changed to just fallback to the default docstring in this case